config/remote/remote_config.go

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package remote_config
+
+import (
+	"math/big"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/spf13/cast"
+
+	"github.com/mysteriumnetwork/node/tequilapi/client"
+
+	"github.com/mysteriumnetwork/node/config"
+	"github.com/rs/zerolog/log"
+)
+
+var Config = newRemoteConfig()
+
+type remoteConfig struct {
+	config map[string]interface{}
+}
+
+func newRemoteConfig() *remoteConfig {
+	return &remoteConfig{}
+}
+
+func RefreshRemoteConfig(client *client.Client) error {
```

Comment:
Hide the `*client.Client` under an interface and dont import the tequilapi/client here. This would make it more flexible and testable if we ever wanted to do either one of those.